src/components/Candidates/Roles/Roles.js

```diff
@@ -20,7 +21,7 @@ type Props = {
   positionsData: PositionsData
 };
 
-class CandidatesJoinUs extends Component<Props> {
+class Roles extends Component<Props> {
```

Comment:
Please convert this to a functional component using Hooks;

Also, eliminate joinUsCallback() if not used